.gitignore

```diff
@@ -9,6 +9,7 @@
 
 # dependencies
 /node_modules
+package-lock.json
```

Comment:
@germanpacken había razón por este cambio?  En general es mejor dejarlo ...

https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Pero igual no importa tanto.